### PR TITLE
WebSockets: add election info section

### PR DESCRIPTION
**Pull request by cryptocode**
https://github.com/nanocurrency/nano-node/pull/2195

### CODE REVIEW

File: docs/integration-guides/advanced.md [hunk 1]
```diff
@@ -290,6 +290,54 @@ To subscribe to all confirmed blocks:
...
+  "options": {
+    "include_election_info": "true"
+  }
+}
+```
+
+Below is a full message which includes the following election info:
+
+* election duration in milliseconds
+* election timestamp as milliseconds since epoch
```
Comment (guilhermelawless) [inline docs/integration-guides/advanced.md]: Can we specify if it's the start or end of the election?

Comment (cryptocode) [reply docs/integration-guides/advanced.md]: Yep, added

**Comment by zhyatt** [comment]
@cryptocode I rearranged the websocket section to include filter options and response options sections to hopefully further clarify things. Can you give it a look?

**Comment by cryptocode** [comment]
@zhyatt That's much better, thanks

**Pull request merged**